tests/test_transformations.py

```diff
@@ -127,6 +160,218 @@ class AllStrict(Mapping):
             expected = {"obj": {"target": 1}}
             self.assertEqual(expected, result)
 
+    def test_all_transformation_strict(self):
+        class User(Schema):
+            name = UnicodeField()
+
+        class Message(Schema):
+            id = IntegerField()
+            user = Subschema(User)
+
+        class Transformed(Schema):
+            user_name = UnicodeField()
+            object = ObjectField()
+
+        class SomeNonStrictMapping(Mapping):
+            source_schema = Message
+            target_schema = Transformed
+
+            user_name = Get('user', 'name')
+            object = All(strict=False)
+
+        class SomeStrictMapping(Mapping):
```

Comment:
Can you simplify the test of All by omitting the mappings entirely (i.e. look at the other tests in this class)? Can you rename the test to express exactly what you are testing? This feels integration-y and too wide a test.